tests/map/options.test.ts

```diff
@@ -33,7 +36,7 @@ describe('MapOptions', () => {
         assert(document.body.innerHTML === KARMA_INSERTED_HTML);
     });
 
-    it('scale label for min/max switches between linear and log', () => {
+    it('selected scale label for min/max switches between linear and log', () => {
```

Comment:
Not sure why this change is here, the previous test name was clearer IMO

Reply:
i have changed it to 'selected' so that it aligns with the other tests, where a 'selection' is always made on the HTML select element

Reply:
but seeing that you changed the others to something clearer, then i will revert this one